packages/common/src/projects/fetch.ts

```diff
@@ -122,7 +122,7 @@ export async function enrichProjectSearchResult(
   result.links.self = getItemHomeUrl(result.id, requestOptions);
   let slugOrId = item.id;
   if (item.properties?.slug) {
-    slugOrId = item.properties.slug.replace("|", "::");
+    slugOrId = keywordSlugToUriSlug(item.properties.slug);
   }
```

Comment:
I'd like to wrap this up into a fn like:

```ts
const getItemIdentifier (item: IItem): string => {
  const slug = item.properties?.slug
  return slug
    ? keywordSlugToUriSlug(slug)
    : item.id
}
```

I think we would use that here as well:

https://github.com/ArcGIS/opendata-ui/pull/11993/files#diff-34f8d7a2d5fcba858cc86ce2a4cd5a6eadb17c8ff2a29469c837ebc3d61dce62R28-R32

Reply:
Great idea - added. Also made the slug converters internal only.